CoreScriptsRoot/Modules/BackpackScript3D.lua

```diff
@@ -113,12 +113,16 @@ local function UpdateLayout()
 		end
 	end
 	
-	width = #ToolsList * ICON_SPACING
-	height = ICON_SIZE + HEALTHBAR_SPACE
+	if #ToolsList == 0 then
```

Comment:
so if we have no tools we set the size of the panel to the size of the healthbar? any reason for this?